tasks/src/table-question-answering/data.ts

```diff
@@ -0,0 +1,60 @@
+import type { Task DataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{ 
+			description:"The Fewshot Table dataset consists of tables that naturally occur on the web, that are formatted as few-shot tasks for fine-tuning language models to improve their few-shot performance.",
+			id: "JeremyAlain/123_test",
+		},
+	],
+	demo: {
+		inputs: [
+			{ label: "Table",
+		          filename: "tableQA.jpg",
```

Comment:
I think you can add a table component (you can see an example in tabular-classification task page)